lib/exec.js

```diff
@@ -440,6 +467,9 @@ function executeNone(command, args, options) {
 const executeDispatchTable = {
     none: executeNone,
     firejail: executeFirejail,
+    nsjail: (command, args, options) => needsWine(command)
```

Comment:
personal preference, but I would rather just have:
```javascript
if (needsWine(command))
    return executeFirejail(command, args, options);
```
at the top of `executeNsjail`.

Reply:
Yeah I dithered on that. But it sounded dishonest to call firejail from a function called "exeuteNsjail" :grin: